Scripts/parameters/vehicle_conversion.py

```diff
@@ -0,0 +1,207 @@
+# Variables for converting tons to vehicles for each commodity class
+
+vehicle_conversion = {
+  "marita": {
+    "truck_share": 0.059,
+    "trailer_truck_share": 0.941,
+    "truck_avg_load": 6.35,
+    "trailer_truck_avg_load": 33.32,
+    "empty_share": 0.5
```

Comment:
We have the same kind of stuff in the cost data input file. Could we use that or are they different in some way?

Reply:
Hmm, we actually could fully utilize purpose specific costdatas here. For empty returning vehicle trips we could either add a new variable to the json or simply state that empty share in vehicle calc is empty share of cost calc - 1. 

The only uncertainty I have here is if the vehicle conversion ends up needing new parameters which would require to be then added into costdata json. Then that json would have variables that are not at all cost related.

Reply:
I guess you could leave the non-cost related parameters in this file.